test.cfg.example

```diff
@@ -0,0 +1,39 @@
+[HandleService2]
+
+# the path to the MongoDB executable to use.
+test.mongo.exe=mongod
+
+# whether to use the wired tiger storage engine for MongoDB. 'true' means yes, anything else means
+# no.
+test.mongo.wired_tiger=false
+
+# Where to store temporary files generated during the test.
+test.temp.dir=temp_test_dir
+
+# Whether test files should be discarded after testing is complete.
+# Anything other than 'false' is considered true.
+test.delete.temp.dir=true
+
+# The root url of the auth server.
+auth-service-url=
+
+# The KBase authorization service mkII url.
+auth-url=
+
+# The root url of the shock server.
+shock-url=
+
+# Admin token
+admin-token=
```

Comment:
Ok, now I remember - I don't think the code that uses the admin token (at least in a role that needs admin privs) is tested currently.

That being said, an admin for what? The blobstore?

Reply:
I assume `KBASE_CI_TOKEN` does not have admin right. If my premise is right, then `admin token` var name is misleading given all tests passed without it.

Based on the code, is used to grant readable ACL. See [here](https://github.com/kbase/handle_service2/blob/7798ffe85bfdeb143fc071192ddacebd4a97ffaf/lib/AbstractHandle/Utils/ShockUtil.py#L109)

Reply:
> If my premise is right, then admin token var name is misleading given all tests passed without it

I think the code that requires the token to be an admin isn't tested

Reply:
It tested with mock objects. 

`@patch.object(Handler, "_is_admin_user", return_value=True)`

Reply:
I don't think so

Reply:
The only thing is from the original comment - the cfg.example file should describe what kind of token is needed.

Reply:
👍

Reply:
> Use admin token only for `add_read_acl` operation, otherwise a test token is sufficient

This doesn't describe what kind of admin rights the token needs to have

Reply:
Not sure what you mean. There are different kind of admin rights?

Reply:
yeah, there are many. Generally each service's admin right are separate, so a user (and therefore their tokens) can have catalog admin rights, workspace read admin rights, but not workspace write admin rights or blobstore admin rights

Reply:
:+1: